.ci/install_qemu.sh

```diff
@@ -64,6 +65,25 @@ build_and_install_static_qemu() {
 	uncompress_static_qemu "${KATA_TESTS_CACHEDIR}/${QEMU_TAR}"
 }
 
+# Build QEMU for Confidential Containers.
+build_and_install_qemu_for_cc() {
+	local artifact="qemu"
+
+	case "${qemu_type}" in
+		tdx)
+			artifact="${qemu_type}-${artifact}"
+
+			build_static_artifact_and_install "tdx-tdvf"
```

Comment:
Hmm.  I'm doing something different as part of https://github.com/kata-containers/tests/pull/5065
I think that one can land, but you'll need to adapt the part there in order to keep it here, and I sincerely think you approach here is good!

Reply:
Also, once https://github.com/kata-containers/tests/pull/5065 lands, you should do something similar for Cloud Hypervisor as well.

Reply:
@fidencio here I can replace the `build_static_artifact_and_install` by calling your `.ci/install_tdvf.sh`. Or I can delete `.ci/install_tdvf.sh` ... my opinion is to keep `.ci/install_tdvf.sh` as it can help with debugging (in you want to replace only tdvf and not rebuild/install the whole thing.

I will adjust `.ci/install_cloud_hypervisor.sh` likewise.

Reply:
I'd keep the `.cin/install_tdvf.sh`.